### PR TITLE
feat(quantic): standalone search box

### PR DESCRIPTION
**Pull request by erocheleau**
Related to [SFINT-4014](https://coveord.atlassian.net/browse/SFINT-4014).

Added a quantic standalone searchbox.
Added an example standalone searchbox component to test it.
Modified the searchbox and standalone searchbox to use a lightning-button-icon.

### CODE REVIEW

**Comment by erocheleau** [comment]
Tested locally ✔️ 
Tested in a build-your-own community ✔️ 
Tested in Napili wrapped in an Aura component ✔️ 

image

https://user-images.githubusercontent.com/2488155/129917444-ec9cf224-4f5d-4bcf-b125-104881c83fa8.mp4

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            role="combobox"
+          >
+            <div class="searchbox__container">
+              <div class={searchBoxContainerClass} role="none">
+                <template if:true={withoutSubmitButton}>
+                  <lightning-icon
+                    class="slds-icon slds-input__icon slds-input__icon_left"
+                    icon-name="utility:search"
+                    size="x-small"
+                    title=""
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:19]: the title is it required?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    size="medium"
+                    name="Clear"
+                    title="Clear"
+                    aria-label="Clear"
+                    alternative-text="Clear"
+                  ></lightning-button-icon>
+                </template>
+              </div>
+              <template if:false={withoutSubmitButton}>
+                <button
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:50]: can we use lightning-button instead?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+  NavigationMixin,
+  // @ts-ignore
+} from 'lightning/navigation';
+import {
+  STANDALONE_SEARCH_BOX_STORAGE_KEY
+} from 'c/quanticUtils';
+
+import search from '@salesforce/label/c.quantic_Search';
+
+const KEYS = {
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:18]: can we put this in quanticUtils?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+        redirectionUrl: 'http://placeholder.com',
+      },
+    });
+    this.unsubscribe = this.standaloneSearchBox.subscribe(() =>
+      this.updateStandaloneState()
+    );
+  }
+
+  disconnectedCallback() {
+    if (this.unsubscribe) {
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:115]: `disconnectedCallback() {
    this.unsubscribe?.();
  }` 
what do you think?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        'aria-labelledby',
+        'fixed-text-label fixed-text-addon-post'
+      );
+    }
+    return CLASS_WITHOUT_SUBMIT;
+  }
+
+  showSuggestions() {
+    this.actualSearchBox.showSuggestions();
+    this.combobox?.classList.add('slds-is-open');
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:182]: beautiful 👍

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js-meta.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+      <target>lightningCommunity__Page</target>
+      <target>lightningCommunity__Default</target>
+  </targets>
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js-meta.xml:8]: indentation

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js-meta.xml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  <targetConfigs>
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js-meta.xml:9]: indentation

File: packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.css [hunk 8]
```diff
@@ -0,0 +1,10 @@
+.page-content {
+    max-width: 1400px;
+    width: 100%;
+    min-width: 800px;
+    min-height: 500px;
```
Comment (jeremierobert-coveo) [inline packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.css:5]: Do we have some css var for those magic numbers ?

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js [hunk 9]
```diff
@@ -30,21 +40,30 @@ export default class QuanticSearchInterface extends LightningElement {
...
+                  pipeline: this.pipeline,
+                },
+              },
+            };
+            setEngineOptions(
+              this.engineOptions,
+              CoveoHeadless.buildSearchEngine,
+              this.engineId,
+              this
+            );
```
Comment (jeremierobert-coveo) [inline packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: Maybe we can put this in a private that returns a promise ?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+    padding-top: 0px;
+    padding-bottom: 0px;
+    border-top-right-radius: 0.25rem;
+    border-bottom-right-radius: 0.25rem;
+    border-width: 0px;
+    box-sizing: border-box;
+    background-color: #005FB2;
+    height: 100%;
+    margin: 0px;
+    width: 46.57px;
```
Comment (jeremierobert-coveo) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:32]: Is there a reason why this value seems too magic ?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css [hunk 11]
```diff
@@ -0,0 +1,42 @@
+.searchbox__container{
+    display: flex;
+    position: relative;
+    border-color: #e5e8e8;
+    height: 48px;
+    box-sizing: border-box;
+    background: #FFFFFF;
```
Comment (jeremierobert-coveo) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:7]: Maybe we can benefit from some css variable for the color and magic lenght ?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 12]
```diff
@@ -0,0 +1,298 @@
...
+
+  handleEnter() {
+    const selectedSuggestion = this.suggestionList?.getCurrentSelectedValue();
+    if (this.suggestionsOpen && selectedSuggestion) {
+      this.actualSearchBox.selectSuggestion(selectedSuggestion.rawValue);
+      this.input.blur();
+    } else {
+      this.actualSearchBox.submit();
+      this.input.blur();
+    }
```
Comment (jeremierobert-coveo) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:204]:
```suggestion
    if (this.suggestionsOpen && selectedSuggestion) {
      this.actualSearchBox.selectSuggestion(selectedSuggestion.rawValue);
    } else {
      this.actualSearchBox.submit();
    }
    this.input.blur();
```

File: packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.html [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+    <div class="page-content">
+        <header class="page-header slds-grid slds-grid_align-center slds-gutters_direct-x-large content-grow">
+            <c-quantic-standalone-search-box engine-id="example-search" class="slds-col flex-grow" redirect-url="http://localhost:3333/preview/c/exampleSearch"></c-quantic-standalone-search-box>
+        </header>
+
+        <main class="slds-grid">
+            <div class="slds-text-heading_large slds-col slds-size_1-of-1 slds-align_absolute-center">Main content</div>
+        </main>
+    </div>
+
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.html:11]: extra space here

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js [hunk 14]
```diff
@@ -1,4 +1,4 @@
-import {LightningElement, api, track} from 'lwc';
+import { LightningElement, api, track } from 'lwc';
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:1]: prettier seems to add these spaces. All the other files don't currently implement this rule. It's pretty annoying though. We should either change the prettier config or run it on all the quantic files and be done with it :P

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js [hunk 15]
```diff
@@ -26,7 +26,6 @@ export default class QuanticSearchBox extends LightningElement {
     suggestions: [],
     value: '',
   };
-
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:29]: Idk. Let's keep this newline here; no?

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js (hunk 9 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: Pas fou ca. Would clean it up a bit

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js [hunk 16]
```diff
@@ -54,24 +73,40 @@ export default class QuanticSearchInterface extends LightningElement {
     }
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:73]:
```suggestion
this.unsubscribeUrlManager?.()
```

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js [hunk 17]
```diff
@@ -54,24 +73,40 @@ export default class QuanticSearchInterface extends LightningElement {
...
+   */
   initialize = (engine) => {
     if (!this.disableStateInUrl) {
       this.initUrlManager(engine);
     }
-    engine.executeFirstSearch();
-  }
+
+    if (!this.skipFirstSearch) {
+      const data = window.localStorage.getItem(
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:86]:
```suggestion
const analyticsData = window.localStorage.getItem(STANDALONE_SEARCH_BOX_STORAGE_KEY);

if (!analyticsData) {
  engine.executeFirstSearch();
  return;
}
window.localStorage.removeItem(STANDALONE_SEARCH_BOX_STORAGE_KEY);
const {analytics} = JSON.parse(data);
engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
```

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 10 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:32]: It is, kinda. It's the value found in the Figma mock ups. There's the same css in quanticSearchBox.css

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html (hunk 2 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:50]: I'm not sure because I didn't make this initially. But it's possible the lightning-buttons come with their own styling we'd have to fight against. Not sure though

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 18]
```diff
@@ -0,0 +1,298 @@
...
+
+export default class QuanticStandaloneSearchBox extends NavigationMixin (
+  LightningElement
+) {
+  
+  labels = {
+    search,
+  };
+
+  // Public props
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:36]: I like this. Maybe a JSDoc compatible notation would be even more helpful for us when we look into implementing it.

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 19]
```diff
@@ -0,0 +1,298 @@
...
+  /** @type {number} */
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:37]: Maybe something like this

```suggestion
/**
 * The maximum number of query suggestion to display.
 * @type {number}
 * @public
 */
```

Maybe this is out of scope though. Also maybe this exact syntax doesn't work.

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 4 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:115]:
```suggestion
disconnectedCallback() {
  this.unsubscribe?.();
  this.unsubscribeSearchBox?.();
}
```

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 20]
```diff
@@ -0,0 +1,298 @@
...
+    this.input.focus();
+  }
+
+  handleSuggestionSelection(event) {
+    const textValue = event.detail;
+    this.actualSearchBox.selectSuggestion(textValue);
+  }
+
+  navigateToSearchPage() {
+    // Navigate to the search page
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:256]: dunno if this is necessary. Your function name is good enough IMO

**Review by nathanlb — COMMENTED** [review]
Really great stuff. A couple things I want your opinion on before approving :)

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 4 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:115]: I had no idea we could do that :O

File: packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.css (hunk 8 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.css:5]: Not for this, it's not really meant to be "used" in a community or anything. It's mainly for us to be able to show the standalone searchbox in localdev server.

Otherwise if you just put the standalone the whole content area of the localdev is only as big as the searchbox input which means we get a scrollbar to see query suggestions, etc.

So I made that one kinda like the exampleSearch. So I wouldn't worry too much about having this "scale" or be editable easily anything like that, what do you think?

I was unsure even to include this component in the PR.

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 10 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:32]: yea this is a copy of quanticSearchBox.css

Nothing should be different AFAIK

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js (hunk 14 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:1]: I think we should set a commit hook to run prettier :) Even if I don't agree with some of the rules.

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js (hunk 9 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: Not sure I understand what you are suggesting, which could be returning a promise?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 11 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:7]: What do you think, should we share css between the searchboxes?
https://help.salesforce.com/s/articleView?id=release-notes.rn_lwc_css_share.htm&type=5&release=226

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html (hunk 1 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:19]: I guess not, this is a copy from quanticSearchBox.html

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html (hunk 2 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:50]: I didn't try for the searchbox because we have more styling I think than on the clear button.

We could look at it in the future I guess?

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 11 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css:7]: Oh yes!

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html (hunk 2 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:50]: image
Just tried it, will update the PR

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 3 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:18]: sure

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 18 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:36]: Oh yea for sure!

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 19 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:37]: Yea I would say very good idea, but out of scope :D

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js [hunk 21]
```diff
@@ -0,0 +1,290 @@
...
+    search,
+  };
+
+  // Public props
+  /** @type {number} */
+  @api numberOfSuggestions = 5;
+  /** @type {string} */
+  @api placeholder = `${this.labels.search}...`;
+  /** @type {string} */
+  @api engineId;
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:35]: there's something weird going on with this file

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html (hunk 1 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html:19]: If it is indeed unnecessary could you remove it from the normal searchbox as well?

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js [hunk 22]
```diff
@@ -5,13 +5,10 @@ import {
   initializeWithHeadless,
 } from 'c/quanticHeadlessLoader';
 
+import { keys } from 'c/quanticUtils';
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:8]: prettier difference again :$

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js (hunk 9 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: I think the suggestion is just extracting this getting and setting of options into a private function. Could improve legibility.

**Review by nathanlb — APPROVED** [review]
I think there are still little comments to address but GG

**Review by aelfaleh — APPROVED** [review]
beautiful good job ❤️

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js (hunk 9 above)
Comment (jeremierobert-coveo) [reply packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: more improving readability than legibility but yeah

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 10 above)
Comment (jeremierobert-coveo) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css]: So it's the new golden number ?

**Review by jeremierobert-coveo — APPROVED** [review]
Only small refactors remaining. Overrall this is awsome

File: packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js (hunk 9 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js:60]: Yes wrong word. My bad

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 10 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css]: We can probably safely round it up

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js (hunk 22 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:8]: Does it really matter? Until we have a commit hook or a prettierjs config file we'll be getting those ;)

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js (hunk 21 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js:35]: you mean github coloring it? Not much I can do about it ;)

File: packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css (hunk 10 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css]: no, in this project we have sub-pixel accuracy!
